Pipfile

```diff
@@ -10,7 +10,6 @@ deprecation = ">=2.1.0"
 pytest = ">=2.7.2"
 PyYAML = ">=5.1"
 rdflib = ">=6.0.0"
-vss-tools = {editable = true, path = "."}
```

Comment:
this turns the pipfile usage useless... since the current project wouldn't be reached from inside pipenv.

To me the problem seems to be https://github.com/COVESA/vss-tools/blob/master/.github/workflows/buildcheck.yml#L24 remove that line.

I'd also suggest fixing the workflow to use https://github.com/actions/setup-python and remove all `pipenv install` handling, also the `PIPENV_VENV_IN_PROJECT` and `PIPENV_PIPFILE`, just say `cache: pip`.

Reply:
Thanks for the ideas. Does not seem to be as easy as just removing line 24.

Reply:
tomorrow I'll try to give it a try and propose a PR fixing this issue and using GitHub's best practices (I'm doing similar stuff for some internal projects, so I got to understand some pitfalls and pain points...)